src/components/Modals/ProjectCardModal/ProjectCardModal.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from 'react';
+
+interface PropTypes {
+  open: boolean;
+  onClose: () => void;
+  onViewMoreProjects: () => void;
+}
+
+const ProjectCardModal: React.FC<PropTypes> = ({ open, onClose, onViewMoreProjects }) => {
+  const [answers, setAnswers] = useState<any[]>([
+    { id: 1, answerOne: "Sample answer one", answerTwo: "Sample answer two" }
+  ]);
+  const [answerOne, setAnswerOne] = useState<string>('');
+  const [answerTwo, setAnswerTwo] = useState<string>('');
+  const [isReviewMode, setIsReviewMode] = useState<boolean>(false);
+  const [submissionComplete, setSubmissionComplete] = useState<boolean>(false);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!answerOne || !answerTwo) {
+      alert("Please answer all the questions.");
+      return;
+    }
+    setIsReviewMode(true);
+  };
+
+  const handleFinalSubmit = () => {
+    const item = {
+      id: Math.floor(Math.random() * 1000),
+      answerOne,
+      answerTwo,
+    };
+    setAnswers((oldList) => {
+      const answers = [...oldList, item];
+      localStorage.setItem("answers", JSON.stringify(answers));
+      return answers;
+    });
+    setAnswerOne("");
+    setAnswerTwo("");
+    setIsReviewMode(false);
+    setSubmissionComplete(true);
+  };
+
+  useEffect(() => {
```

Comment:
I know you are trying to save data for submissions. I really don't think this is the right way long term. I would move on to another way of storing answers (which is tricky for you since the backend is out of your control). Reason being is if you start saving all this data locally, you could get it stolen or it can disappear if you are not using the same machine.

Also maybe a cookie with a lifetime would be better so that it's not forever on your machine.

Reply:
Agreed. Since this is temporary until the backend part gets figured out I will leave it for now, but I added a TODO comment to make sure to find a better way.